RoaringBitmap/src/main/java/org/roaringbitmap/Container.java

```diff
@@ -349,6 +349,13 @@ public String getContainerName() {
    */
   public abstract void forAllFrom(char startValue, final RelativeRangeConsumer rrc);
 
+  /*
+   * Design Note
```

Comment:
Not a request for change: We could have an inclusive "forAllUpToAndIncluding" where you would just branch on the function you want. I am not asking you to do that.

Reply:
We could, but it would have to take a lot of logic (and needed parameters) from the upper (RoaringBitmap) layer, so I'm not sure that would actually improve the code readability here.